docs/plugins/repository-gcs.asciidoc

```diff
@@ -88,8 +88,9 @@ A JSON service account file looks like this:
 ----
 // NOTCONSOLE
 
-To provide this file to the plugin, it must be stored in the {ref}/secure-settings.html[Elasticsearch keystore].  You must add a setting name of the form `gcs.client.NAME.credentials_file`, where `NAME`
-is the name of the client configuration for the repository. The implicit client
+To provide this file to the plugin, it must be stored in the {ref}/secure-settings.html[Elasticsearch keystore].  You must
```

Comment:
Not blocking but we can use the `{es}` attribute instead of `Elasticsearch` here.

Reply:
I'm not sure what the `{es}` attribute is. Can you suggest a specific change to the text?

Reply:
`{es}` here will expand to `Elasticsearch`.

```suggestion
To provide this file to the plugin, it must be stored in the {ref}/secure-settings.html[{es} keystore].  You must
```

Reply:
I'm going to skip this for now, as there are other uses of "Elasticsearch" in this file and I don't want to create an inconsistency.